src/js/pages/Ballot/Ballot.jsx

```diff
@@ -1381,7 +1381,7 @@ class Ballot extends Component {
     let numberOfBallotItemsDisplayed = 0;
     let showLoadingText = true;
     let searchTextString = '';
-    const showCompleteYourProfile = isWebApp();
+    const showCompleteYourProfile = true;
```

Comment:
Hi @itcreativeusa anytime you see `isWebApp()` we want to leave that in place because is what our mobile developer uses to indicate he doesn't wan't it to show in our Cordova App Store apps.